apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -404,82 +406,166 @@ onDocumentReady(() => {
-                  <option value="-1">Subtract from instances with time limit</option>
-                `
-              : html`
-                  <option value="+1">Add</option>
-                  <option value="-1">Subtract</option>
-                `
-            : ''}
+          ${
+            row.time_remaining_sec !== null
+              ? row.has_open_instance
+                ? html`
+                    <option value="+1">Add to instances with time limit</option>
+                    <option value="-1">Subtract from instances with time limit</option>
+                  `
+                : html`
+                    <option value="+1">Add</option>
+                    <option value="-1">Subtract</option>
+                  `
+              : ''
+          }
           <option value="set_total">Set total time limit to</option>
           <option value="set_rem">Set remaining time to</option>
-          ${row.open || row.time_remaining_sec !== null
-            ? html`<option value="unlimited">Remove time limit</option>`
-            : ''}
-          ${row.open !== false && (row.time_remaining_sec === null || row.time_remaining_sec > 0)
-            ? html`<option value="expire">Expire time limit</option>`
-            : ''}
+          <option value="set_exact">Set exact closing time</option>
+          ${
+            row.open || row.time_remaining_sec !== null
+              ? html`<option value="unlimited">Remove time limit</option>`
+              : ''
+          }
+          ${
+            row.open !== false && (row.time_remaining_sec === null || row.time_remaining_sec > 0)
+              ? html`<option value="expire">Expire time limit</option>`
+              : ''
+          }
         </select>
-        <p class="form-inline">
-          <input
-            class="form-control time-limit-field"
-            type="number"
-            name="time_add"
-            aria-label="Time value"
-            style="width: 5em"
-            value="5"
-          />
-          <select class="custom-select time-limit-field" name="time_ref" aria-label="Time unit">
-            <option value="minutes">minutes</option>
-            ${row.time_remaining_sec !== null
-              ? html`<option value="percent">% total limit</option>`
-              : ''}
-          </select>
+        <small class="small form-text text-muted add">This will add time to the total time limit.</small>
+        <small class="small form-text text-muted subtract" style="display:none">This will subtract time from the total time limit.</small>
+        <small class="small form-text text-muted total-time" style="display:none">This will set the total time limit to the given amount.</small>
+        <small class="small form-text text-muted remaining-time" style="display:none">This will set the remaining time limit to the given amount.</small>
+        <small class="small form-text text-muted date-picker" style="display:none">This will set the exact closing time.</small>
```

Comment:
I know I originally suggested adding explanations, but I think many of these don't actually add much:

<img width="296" alt="Screenshot 2024-10-07 at 16 16 43" src="https://github.com/user-attachments/assets/8a6b2b91-7ae5-40cc-be3c-145c281082c5">
<img width="351" alt="Screenshot 2024-10-07 at 16 16 36" src="https://github.com/user-attachments/assets/32650570-c7a9-4cba-8ee0-e5b9856b50d9">

These seem to just be restating what's already in the `<select>` options.